### PR TITLE
Add reward activity

### PR DESCRIPTION
**Pull request by izzyalonso**

### CODE REVIEW

File: src/main/java/org/tndata/android/compass/model/Reward.java [hunk 1]
```diff
@@ -83,6 +123,38 @@ public SpannableString format(){
...
+    /**
+     * Shares this reward via Intent.
+     *
+     * @param context a reference to the context.
+     */
+    public void share(Context context){
+        //Build the content string
+        String content = "";
+        if (isJoke()){
+            content = "Joke: ";
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/model/Reward.java]: Do we want to put these `content` strings in `strings.xml`?

File: src/main/java/org/tndata/android/compass/model/Reward.java [hunk 2]
```diff
@@ -83,6 +123,38 @@ public SpannableString format(){
...
+        content += getMessage();
+        if (isQuote()){
+            content += "\"";
+        }
+
+        //Send the intent
+        Intent shareIntent = new Intent(android.content.Intent.ACTION_SEND);
+        shareIntent.setType("text/plain");
+        shareIntent.putExtra(android.content.Intent.EXTRA_TEXT, content);
+        context.startActivity(Intent.createChooser(shareIntent, "Share via"));
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/model/Reward.java]: Should the "Share via" go in `strings.xml`, too?

Comment (izzyalonso) [inline src/main/java/org/tndata/android/compass/model/Reward.java]: Sure

File: src/main/java/org/tndata/android/compass/model/Reward.java (hunk 1 above)
Comment (izzyalonso) [inline src/main/java/org/tndata/android/compass/model/Reward.java]: Yeah

**Pull request merged**